src/storage/mvcc/reader/scanner/backward.rs

```diff
@@ -223,19 +236,52 @@ impl<S: Snapshot> BackwardKvScanner<S> {
         // At this time, we must have current commit_ts <= ts. If commit_ts == ts,
         // we don't need to seek any more and we can just utilize `last_version`.
         if last_checked_commit_ts == ts {
+            self.write_cursor.prev(&mut self.statistics.write);
+            if let Ok(true) = self.write_cursor.valid() {
```

Comment:
I think errors should be thrown to outside:

```rust
if self.write_cursor.valid()? {
```

Reply:
Addressed

Reply:
When the prev write_cursor is not valid，I think we should not thrown because the current (not prev) data at write_cursor is good.
When the prev write_cursor is not valid，It means no newer data of this user-key, aka, cache_able